pyodide-test-runner/pyodide_test_runner/decorator.py

```diff
@@ -134,10 +131,6 @@ def __init__(
         packages : List[str]
             List of packages to load before running the test
 
-        driver_timeout : Optional[float]
-            selenium driver timeout (in seconds). If missing, use the default
-            timeout.
```

Comment:
Could we document the marks somewhere, maybe in https://pyodide.org/en/latest/development/testing.html ?

As with https://github.com/pyodide/pyodide/blob/main/setup.cfg#L13 even if one find the right file where they are listed, it doesn't really say how to use it.

Reply:
I will try to update the testing docs in a separate PR. Currently I have adjusted the docs in #2617 so there will be a merge conflict if I update it here too.